README.md

```diff
@@ -86,6 +86,15 @@ cuGraph provides an auto-renumbering feature, enabled by default, during Graph c
 
 cuGraph is constantly being updatred and improved. Please see the [Transition Guide](TRANSITIONGUIDE.md) if errors are encountered with newer versions
 
+## Graph Sizes and GPU Memory Size

```

Comment:
(Irrelevant to the line above).

In
https://github.com/rapidsai/cugraph/pull/859/files#diff-04c6e90faac2675aa89e2176d2eec7d8R5

For users familar with C/CUDA and graph structures, a C++ API is also provided.  However, there is less type and structure checking at the C layer.
=>

For users famil"i"ar with C"++"/CUDA and graph structures, a C++ API is also provided.  However, there is less type and structure checking at the C"++" layer.

Reply:
fixed